app/scripts/metamask-controller.js

```diff
@@ -1518,6 +1520,17 @@ export default class MetamaskController extends EventEmitter {
       state: initState.MetamaskNotificationsController,
     });
 
+    // Temporary add missing methods (due to notification controller migration)
+    this.controllerMessenger.registerActionHandler(
+      'NotificationServicesController:disableNotificationServices',
+      () => this.metamaskNotificationsController.disableMetamaskNotifications(),
+    );
+    this.controllerMessenger.registerActionHandler(
+      'NotificationServicesController:selectIsNotificationServicesEnabled',
+      () =>
+        this.metamaskNotificationsController.selectIsMetamaskNotificationsEnabled(),
+    );
```

Comment:
We are slowly migrating controllers to use the shared controllers.
This addition is so that this PR works standalone (there was some controller renaming that was done in shared libraries)